src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlDependency.cs

```diff
@@ -305,11 +380,17 @@ public SqlDependency(SqlCommand command, string options, int timeout)
                         {
                             if (_dependencyFired)
                             { // If fired, fire the new event immediately.
+#if NETFRAMEWORK
```

Comment:
Should these events be netfx only? I think it was accidentally left out from netcore.

Reply:
I don't think it affects netcore since it's only adds extra logging, I'll remove the ifdef and update the string format to keep it consistent in the next commit.